spec/features/tags/user_tags_new_api_spec.rb

```diff
@@ -20,14 +25,18 @@
 
   scenario "adds api with no tags" do
     visit root_path
+    click_link 'Sign In'
+    fill_in 'Email', with: user1.email
+    fill_in 'Password', with: user1.password
+    click_button 'Sign In'
 
+    visit root_path
     click_link "Add New"
     fill_in("Name", with: "apiai")
     fill_in("URL", with: "www.api.ai")
     fill_in("Description", with: "Right side of Gabby's brain.")
     choose("Yes.")
     fill_in("Tags", with: "")
-
     click_button "Create API"
     api = Api.where(name: "apiai").first
```

Comment:
This stuff is testing the database, which should only be in a unit test. Factor it out. Use an expect statement where you expect to see the api info on the page, and don't do things where you check that something in the db is `nil`.